pinery-lims/src/main/java/ca/on/oicr/pinery/lims/SampleAttribute.java

```diff
@@ -29,7 +29,7 @@
     SAMPLE_RECEIVE_DATE("geo_receive_date", "Receive Date"),
     EXTERNAL_NAME("geo_external_name", "External Name"),
     TUBE_ID("geo_tube_id", "Tube ID"),
-    TARGETED_RESEQUENCING("geo_targeted_resequencing", "Targeted Resequencing"),
+    TARGETED_RESEQUENCING("geo_targeted_resequencing", "Targeted Resequencing", "Targeted Sequencing"),
```

Comment:
pinery-lims-miso (MisoClient) maps "targeted sequencing" to "targeted resequencing" here: https://github.com/oicr-gsi/pinery/blob/GP-972_2.0-rc6/pinery-lims-miso/src/main/java/ca/on/oicr/pinery/lims/miso/MisoClient.java#L1145 making the SampleAttribute "Targeted Sequencing" map not needed.  I'll remove that entry.